ui/src/main/webapp/src/app/project/migration-project.service.ts

```diff
@@ -42,7 +46,8 @@ export class MigrationProjectService extends AbstractService {
 
         if (event.isTypeOf(NewExecutionStartedEvent)) {
             monitoredProject.executions.push((<NewExecutionStartedEvent>event).execution);
-        } else if (event.isTypeOf(ExecutionUpdatedEvent)) {
+        }
+        else if (event.isTypeOf(ExecutionUpdatedEvent)) {
```

Comment:
Sorry Ondra, but I have completely different point of view on this.

We have some code style, which everyone follows (or at least mostly, we probably have some older parts of code which are not consistent). And those formatting changes are not only matter of this PR,  so it is not about us having a release. It is about having consistent code base. 

You might have noticed I comment these on quite a lot of your PRs. It is not about trolling you. It is to point out you are sometimes incorrectly changing formatting. Every time, I point it out, I hope you won't do it next time. But you do. So I point it out again. 

What should I do? Ignore it? Sure, I could do that. But what's the point of having some code style then? So should we stop caring about code style? I don't think so. 

But I have a suggestion - lets do huge formatting fix PR after release and set up some code style checker running before tests. It will force everyone to follow consistent code style and it won't be necessary to argue about it in each PR.